spec/requests/stories/feeds_spec.rb

```diff
@@ -4,27 +4,55 @@
   let(:title) { "My post" }
   let(:user) { create(:user, name: "Josh") }
   let(:organization) { create(:organization, name: "JoshCo") }
-  let!(:article) { create(:article, title: title, featured: true, user: user, organization: organization) }
+  let(:tags) { "alpha, beta, delta, gamma" }
+  let(:article) { create(:article, title: title, featured: true, user: user, organization: organization, tags: tags) }
+
+  before do
```

Comment:
Would it be easier to use `let!(:article)`, which forces method invocation before each example? And if we don't need to re-create the article before each spec, you can use the [`let_it_be`](https://github.com/palkan/test-prof/blob/master/docs/let_it_be.md) helper like we do in other places. 😃

Reply:
I had no idea about `test_prof` and `let_it_be`, thanks for the heads up!

I actually do want to recreate the article before each spec (or at least before each context): there are different options we want to have on the article for each context. The issue with using `let!` is sequencing with the setting of other attributes we're setting on article.

Reply:
Thanks for clarifying @joshpuetz 😃